packages/cli/src/utils.ts

```diff
@@ -132,14 +132,32 @@ const checkImageExtension = (uri: string): boolean => {
   );
 };
 
+export const isDevnet = (rpcUrl: string): boolean => {
+  return rpcUrl.indexOf("devnet") != -1;
+};
+
+export const isTestnet = (rpcUrl: string): boolean => {
+  return rpcUrl.indexOf("testnet") != -1;
+};
+
+export const isMainnet = (rpcUrl: string): boolean => {
```

Comment:
I don't know that this heuristic is accurate - I'd propose that mainnet is the default, in the absence of a signal that it is NOT mainnet

Reply:
Can do.